product-development/README.md

```diff
@@ -199,6 +199,17 @@ Here is an example of how an epic description should look when all the acceptanc
 
 Between the end of one epic and the start of a new one, it is OK to knock out small feature requests or [bug fixes](https://github.com/tilt-dev/company/blob/master/development/README.md#assign-yourself-a-bug) that you have context on.
 
+## Moving Between Teams
+
+Since we are focused on shipping prioritized epics quickly, Nick may
+move you to another team to work on another epic, even before the current
+epic you are working on is done. Feel free to make suggestions. Nick
```

Comment:
I don't want to propose any additional process at this point, but we should be cognizant of how frequently this happens and the resulting impact on the de-prioritized epic work. (Specifically, if we _don't_ return to that epic afterwards, we might at least want to ensure that we do some sort of wrap-up, whether that's rolling back any incomplete feature code or ensuring that adequate context is present in CH to resume work.)

Reply:
Thanks @milas . The intention here is that if you move to another team, the old team you were on will continue to drive the epic to completion. I.e. we certainly don't want to have paused unfinished epics, as much as possible. I see how my wording here doesn't make that clear. I've tried to improve it. Thanks for calling this out!